src/scenes/ApplicationRouter/portal/Review/index.tsx

```diff
@@ -88,12 +122,20 @@ const ReviewView = () => {
     [applicationSections]
   );
 
-  return (
-    <ApplicationPage crumbs={crumbs}>
-      {selectedApplication?.status === 'In Review' && <ApplicationStatusSubmitted />}
-      {selectedApplication?.status !== 'Submitted' && <ReviewCard sections={nonPrescreenSections} />}
-    </ApplicationPage>
-  );
+  const renderContent = (application: Application | undefined) => {
+    switch (application?.status) {
+      case 'In Review':
+        return <ApplicationStatusInReview />;
+      case 'Not Accepted':
+        return <ApplicationStatusNotAccepted application={application} />;
+      case 'Waitlist':
```

Comment:
I think we need to add the Issue Active, Issue Pending, Issue Resolved cases here. 

Not common but it will show up incorrect for a Global admin since they will receive detail statuses instead of external ones.

Reply:
Do we have copy and/or designs for those application states?

Reply:
Those all count as Waitlist

Reply:
Got it, will update. Thanks @tommylau523! 🙌